docs/demos/ChatBoot/style.css

```
@@ -0,0 +1,60 @@
+.sidebar-left {
+  width: 256px;
+  min-width: 256px;
+  max-width: 256px;
+  padding: 0 16px 16px 16px;
+  transition: opacity 500ms ease-in-out;
+  overflow: hidden;
+  box-sizing: border-box;
+  padding: 12px;
+  /* gray/gray-边线-浅 */
+  /* 样式描述：--gray-a3 */
+  border-right: 1px solid var(--color-gray-border-light);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)